### ISSUE TITLE
High DPI Support

### ISSUE THREAD

**Issue by Infernio** [wrye-bash/wrye-bash#555]
Randomly got curious today and wanted to see just how we behave with a >100% DPI scaling factor. The answer is badly:

![image](https://user-images.githubusercontent.com/2845240/102543146-280a0100-40b3-11eb-80f8-f23e49283be2.png)

Might be a little hard to see (clicking to enlarge it may help), but the entire WB app is blurry (i.e. not high DPI-aware at all). This would have been almost impossible to solve (or at least would have requried tons of hacks) on older wxPython versions, but wxPython 4.1 (see #553) gives us the tools we need for this.
wxPython docs for this are here: https://wxpython.org/Phoenix/docs/html/high_dpi_overview.html

- [x] Blocked on #553
- [x] We need a set of high-quality versions of our in-GUI images -> new issue: #557

**Comment by Infernio** [comment]
Reassigned to 308, worst case we'll ship this without the high res images and it'll still be very usable (and a vast improvement over the current blurry state).
